src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -231,9 +231,10 @@ function processInlineTag(
   currentStyle: DraftInlineStyle,
 ): DraftInlineStyle {
   var styleToCheck = inlineTags[tag];
+  var win = node.ownerDocument.defaultView || window;
   if (styleToCheck) {
     currentStyle = currentStyle.add(styleToCheck).toOrderedSet();
-  } else if (node instanceof HTMLElement) {
+  } else if (node instanceof win.HTMLElement) {
```

Comment:
node.nodeType === 1

Reply:
In this case, refining the node type to `Element` is insufficient, because the `style` object is a part of the `HTMLElement` interface. I can make Flow be ok with that with:

```js
const htmlElement: HTMLElement = (node: any);
```

But seems strictly speaking to be inaccurate?